htdocs/install/mysql/migration/20.0.0-21.0.0.sql

```diff
@@ -40,6 +40,10 @@ ALTER TABLE llx_projet_task ADD COLUMN billable smallint DEFAULT 1;
 
 ALTER TABLE llx_inventory DROP COLUMN datec;
 
+DELETE FROM llx_document_model WHERE nom='standard' AND type='expensereport';
```

Comment:
Delete or update with the new name ?

Reply:
I tried first with rename, but on migration module reload create the new entry